src/EnergyPlus/Photovoltaics.hh

```diff
@@ -135,8 +135,8 @@ namespace Photovoltaics {
                 Real64 &XX,
                 std::function<Real64(EnergyPlusData &state, Real64 const, Real64 const, Real64 const, Real64 const, Real64 const, Real64 const)> FXX,
                 std::function<Real64(EnergyPlusData &state, Real64 const, Real64 const, Real64 const, Real64 const, Real64 const)> DER,
-                Real64 const &II, // Autodesk Aliased to XX in some calls
-                Real64 const &VV, // Autodesk Aliased to XX in some calls
+                Real64 const &II, // Autodesk Aliased to XX in some calls, this is absolutely wacky and needs to go
```

Comment:
I know that I said that passing scalar parameters by const ref is a bad idea but I am leaving this here for now. The reason is that in some calls of this function the same variable is passed through two parameters, via `XX` by reference and via `VV` or `II` by const reference. The function relies on the fact that when the variable is changed via `XX`, the change is reflected in `VV` or `II`. Because of this possibility,  expressions using `VV` and `II` cannot be optimized. And they could not be optimized even if the function was never called with the same variable used for multiple reference arguments. This function needs to be rewritten to avoid this behavior.